libnmstate/nm/route.py

```diff
@@ -37,52 +36,6 @@
 ROUTE_RULE_DEFAULT_PRIORIRY = 30000
 
 
-def get_running_config(applied_configs):
-    """
-    Query routes saved in running profile
-    """
-    routes = []
-    for iface_name, applied_config in applied_configs.items():
-        for ip_profile in (
-            applied_config.get_setting_ip6_config(),
-            applied_config.get_setting_ip4_config(),
-        ):
-            if not ip_profile:
-                continue
-            nm_routes = ip_profile.props.routes
-            gateway = ip_profile.props.gateway
-            if not nm_routes and not gateway:
-                continue
-            default_table_id = ip_profile.props.route_table
-            if gateway:
-                routes.append(
-                    _get_default_route_config(
-                        gateway,
-                        ip_profile.props.route_metric,
-                        default_table_id,
-                        iface_name,
-                    )
-                )
-            # NM supports multiple route table in single profile:
-            # https://bugzilla.redhat.com/show_bug.cgi?id=1436531 The
-            # `ipv4.route-table` and `ipv6.route-table` will be the default
-            # table id for static routes and auto routes. But each static route
-            # can still specify route table id.
-            for nm_route in nm_routes:
-                table_id = _get_per_route_table_id(nm_route, default_table_id)
-                route_entry = _nm_route_to_route(
-                    nm_route, table_id, iface_name
-                )
-                if route_entry:
-                    routes.append(route_entry)
-    routes.sort(
-        key=itemgetter(
-            Route.TABLE_ID, Route.NEXT_HOP_INTERFACE, Route.DESTINATION
-        )
-    )
-    return routes
-
-
 def _get_per_route_table_id(nm_route, default_table_id):
     table = nm_route.get_attribute(NM.IP_ROUTE_ATTRIBUTE_TABLE)
     return int(table.get_uint32()) if table else default_table_id
```

Comment:
This function is not used anymore, please remove it. The same for the function `_nm_route_to_route` and `_get_default_route_config`. Thanks!

Reply:
Thanks. Removed.